package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "arweave",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Should I add more functions before jumping to a minor? I know we have other calls that we don't have functions for them on arweave-js yet. Or do we go to 1.5.0 and then update with other functions?

Reply:
Yeah good point, if there's other missing functions we can easily add lets add them in first and then jump up to 1.5

Reply:
Lets just increment it with each new function that gets merged to release, no harm in having a large minor on the version number. What other functions are missing?

Reply:
Yes, @arweave-kyle good idea and since it's not a patch, it makes more sense. Thank you. I think I saw something else missing but now I can't remember. We can publish this update and I will take a look again during next week 👍 Thank you!